### DISCUSSION TITLE
New trigger based on post meta

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-future#852] [Feature Requests]
Related: https://github.com/publishpress/PublishPress-Future/issues/732

We could consider a trigger that starts when something changes on a post. For example, when there's a particular piece of metadata added to the post.

One customer example: "Can use I the plugin with ACF so when a user fills in the fields, I can add custom actions every action can send email to employee'

**Comment by andergmartins** [comment]
Another request:

>I want to use an ACF date field value to update the Publish date IF the date field value is in the future. Not all the posts will have future dates. Any ideas on how i can do this ?
The ACF field is scheduled_publish_date_time and this is on custom post type not standard post.
